docs/tutorial_1/1_beam_introduction.ipynb

```diff
@@ -1,876 +1,1681 @@
+    "\n",
+    "The dimension of the archive is `638 MB`, which means it may take some time to download and uncompress. (⚠️ **this may take sometime, depending on your internet connection**) Please keep this in mind while you will run the next cell."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title Run this cell to initiate the download { display-mode: \"form\" }\n",
+    "if not len(os.listdir(DATASET_FOLDER)) or not os.path.exists(DATA_FILE):\n",
+    "    # Checking here to avoid download multiple times in case of notebook re-run\n",
+    "    kaggle.api.dataset_download_files(\"netflix-inc/netflix-prize-data\", \n",
+    "                                      path=DATASET_FOLDER, unzip=True, quiet=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#@title **Inspect** the content of the Data folder { display-mode: \"form\" }\n",
+    "os.listdir(DATASET_FOLDER)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### `Netflix-prize` data files\n",
+    "\n",
+    "Before getting into the actual processing of this dataset with **Apache Beam**, let's briefly describe the actual data files included in the Kaggle archive. \n",
+    "\n",
+    "The dataset is organised into multiple files, accounting for over `100` million ratings from `480` thousand\n",
+    "randomly-chosen, anonymous Netflix customers, over `17` thousand movie titles. \n",
+    "\n",
+    "- The `probe.txt` and the `qualitfying.txt` files are needed for the submission on Kaggle, so discarded as not interesting here.\n",
+    "\n",
+    "- Movie information (i.e. _metadata_) are contained in the `movie_titles.txt` in the following format:\n",
+    "\n",
+    "```\n",
+    "MovieID,YearOfRelease,Title\n",
+    "\n",
+    "- MovieID is a general (randomised) ID for movies, rangin from 1 to 17,770.\n",
+    "- YearOfRelease can range from 1,890 to 2,005.\n",
+    "- Title (in English) is the Netflix title.\n",
+    "```\n",
+    "\n",
+    "- The various `combined_data_*.txt` contain the actual data in the following format:\n",
+    "\n",
+    "```\n",
+    "MovieID:\n",
+    "CustomerID,Rating,Date\n",
+    "CustomerID,Rating,Date\n",
+    "....\n",
+    "\n",
+    "- MovieIDs matches those in the list of \"movie_titles.txt\".\n",
+    "- CustomerIDs range from 1 to 2,649,429, with gaps. There are 480,189 users.\n",
+    "- Ratings are on a five star (integral) scale from 1 to 5.\n",
+    "- Dates have the format YYYY-MM-DD.\n",
+    "```\n",
+    "\n",
+    "A more thorough description of each of those files is available in the [`README`](netflix_prize_dataset/data/README) file. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Preparing our dataset \n",
```

Comment:
My feeling there is too much information for the user who sees this dataset for the first time. What do you think about put it in a hidden by default section with name something like:
"Details on preparing our dataset (optional)"
and to have 1-2 cell to run outside of this cell in order that the user has shortcut ?

Reply:
Sure - I guess you're right! :)